code/datums/actions/order_action.dm

```diff
@@ -11,13 +11,15 @@
 	///What minimum level in that skill is needed to have that action
 	var/skill_min = SKILL_LEAD_TRAINED
 
+	var/cooldown_timer
+
```

Comment:
Is this used?

Reply:
nope. Was going to do the visible countdown, but figured it was scope creep and forgot to delete it.